homeassistant/components/homewizard/sensor.py

```diff
@@ -425,18 +413,133 @@ class HomeWizardSensorEntityDescription(
     ),
 )
 
+EXTERNAL_SENSORS = {
+    ExternalDevice.DeviceType.GAS_METER: SensorEntityDescription(
+        key="Gas meter",
```

Comment:
The keys seem to be natural language, whereas the existing sensors don't have those kind of keys. Why is this? Have things changed upstream since your original PR?

Reply:
I misuse the key in device name ("Gas meter (G0017591236671715)"). 

We now maybe can use the `name` field, but that also feels wrong. Do you maybe have a suggestion?

Reply:
We now allow composing entity names of a translated base and a placeholder.
In this case, you should set the translated name to "Gas meter ({serial_number})", and then set the serial_number in the `_attr_translation_placeholders`
Or this doesn't help you because you use this as base for the device name, not the entity name?
If that's the case, maybe add a `device_name` to `HomeWizardExternalSensorEntityDescription` so you can change it later without affecting the key?

Reply:
Good one, picket the `device_name` works nicely!